From d6eaf640985c6b01928f90386d8fc5c92d6ec836 Mon Sep 17 00:00:00 2001
From: Sandy Huang <hjc@rock-chips.com>
Date: Tue, 19 Mar 2019 10:28:11 +0800
Subject: [PATCH] media-bus: Add SRGB888 media bus format

The output timing described at [1], focus at s888 mode and
s888 dummy mode:

[1] https://patchwork.kernel.org/patch/9992241/

Change-Id: I1bcc6d64ede243d89807acc7e842bcc7fd120c26
Signed-off-by: Sandy Huang <hjc@rock-chips.com>
---
 include/uapi/linux/media-bus-format.h | 8 +++++++-
 1 file changed, 7 insertions(+), 1 deletion(-)

diff --git a/include/uapi/linux/media-bus-format.h b/include/uapi/linux/media-bus-format.h
index 76fd35334192..be55ae7d58fc 100644
--- a/include/uapi/linux/media-bus-format.h
+++ b/include/uapi/linux/media-bus-format.h
@@ -34,7 +34,7 @@
 
 #define MEDIA_BUS_FMT_FIXED			0x0001
 
-/* RGB - next is	0x101c */
+/* RGB - next is	0x1022 */
 #define MEDIA_BUS_FMT_RGB444_1X12		0x1016
 #define MEDIA_BUS_FMT_RGB444_2X8_PADHI_BE	0x1001
 #define MEDIA_BUS_FMT_RGB444_2X8_PADHI_LE	0x1002
@@ -62,6 +62,12 @@
 #define MEDIA_BUS_FMT_RGB121212_1X36		0x1019
 #define MEDIA_BUS_FMT_RGB161616_1X48		0x101a
 #define MEDIA_BUS_FMT_RGB666_1X7X3_JEIDA	0x101b
+#define MEDIA_BUS_FMT_SRGB888_3X8		0x101c
+#define MEDIA_BUS_FMT_SBGR888_3X8		0x101d
+#define MEDIA_BUS_FMT_SRBG888_3X8		0x101e
+#define MEDIA_BUS_FMT_SRGB888_DUMMY_4X8		0x101f
+#define MEDIA_BUS_FMT_SBGR888_DUMMY_4X8		0x1020
+#define MEDIA_BUS_FMT_SRBG888_DUMMY_4X8		0x1021
 
 /* YUV (including grey) - next is	0x202d */
 #define MEDIA_BUS_FMT_Y8_1X8			0x2001
-- 
2.35.3

